### DISCUSSION TITLE
Are there any special latex comments serving as cwl commands?

### DISCUSSION THREAD

**Discussion by PNBRQK** [texstudio-org/texstudio#2625] [Q&A] [answered]
For instance I'd like to tell TS that a macro argument is already math mode text, stopping it from complaining about not-in-math-mode use of superscripts etc.

Searching for the topic gave me a result of #277, which involves playing with a .cwl file. but I created the macro for just one .tex document and do not bother to write a .cwl for it.

Since Vim can recognize information like file encoding from special comments in the source file, can TS do the similar thing for cwl commands from latex source file as well?

**Comment by sunderme** [comment, marked as answer]
no, you need a cwl file for that.

**Reply by PNBRQK** [reply]
Oh yeah, it would be great if something like `% !cwl \mycommand{matharg%formula}`works in latex documents...
